### DISCUSSION TITLE
cloned axis moving at slightly different rates, but only during G0

### DISCUSSION THREAD

**Discussion by wbendick** [fra589/grbl-Mega-5X#368] [Q&A]
My x axis has two steppers moving it, 1 and 4. My y axis has two steppers moving it, 2 and 5. If I perform a homing operation or jog either axis, they move in sync. If I do G0X500, the two steppers will diverge very slightly until the axis starts getting stuck because it has become wedged. The microstepping pins are set the same on all axis. Does anyone have any ideas what could be causing this? It seemed like lost steps, but only during a G0. I tried unplugging my vfd entirely to make sure that wasn't generating any interference, didn't help. Checking the step pins for axis 1 and 4 with an oscilloscope I see one is running at about 15khz while the other at about 13khz

**Comment by fra589** [comment]
Hi @wbendick,  
  
Can you post your specific config.h and the `$I` and `$$` Grbl commands result ?

@++;  
Gauthier.
